### DISCUSSION TITLE
How to add an imprint to footer

### DISCUSSION THREAD

**Discussion by klartextmedia** [mastodon/mastodon#19551] [General]
How can i add an imprint link to the footer and how to setup this static page? Is there any cache that I need to flush to get this modifications live?

**Comment by Gargron** [comment]
Add an imprint section to the "Extended description" field in admin settings. The about page is linked from every page. This is what we do at mastodon.social/mastodon.online. In the future, I intend to add a separate "Imprint" setting.

**Reply by klartextmedia** [reply]
thanks, the problem is that it needs to be called Imprint in germany and it must be on every singe page. The problem is that i have modified some files like app/views/layouts/public.html.haml, featues/ui/components/link_footer.js or even the translation file in javascript/mastodon/locales..... it had no effect so there must be some type of caching, but I can't figure out how to clear this caches.

**Comment by NSFWHubSocial** [comment]
I'm running into the same cache issue. Is there a way to clear the UI cache?

**Reply by CiroArtigot** [reply]
I have the same problem, i don´t know the way to refresh de output after UI changes

**Reply by CiroArtigot** [reply]
I think I have solved the cache issue
These lines in the terminal have solved the problem for me:
`su mastodon`
`RAILS_ENV=production bundle exec rails assets:precompile`
`exit`
`sudo systemctl restart mastodon-*
`

**Comment by apfelnutzer** [comment]
I had the same issue and I adjusted the (german) translation for "Privacy Statement" accordingly and added the imprint information there (did not want to add it to the about section and wanted it to be named properly). Works for now, but I'd appreciate a dedicated imprint link/section as well, so that I do not need to adjust this manually wither every update.
